test/e2e/network/dual_stack.go

```diff
@@ -204,8 +206,152 @@ var _ = SIGDescribe("[Feature:IPv6DualStackAlphaFeature] [LinuxOnly]", func() {
 
 		assertNetworkConnectivity(f, *serverPods, *clientPods, "dualstack-test-client", "80")
 	})
+
+	ginkgo.It("should create service with cluster ip from primary service range [Feature:IPv6DualStackAlphaFeature:Phase2]", func() {
+		serviceName := "defaultclusterip"
+		ns := f.Namespace.Name
+		jig := e2eservice.NewTestJig(cs, serviceName)
+
+		defaultIPFamily := v1.IPv4Protocol
+		if framework.TestContext.ClusterIsIPv6() {
+			defaultIPFamily = v1.IPv6Protocol
+		}
+
+		t := e2eservice.NewServerTest(cs, ns, serviceName)
+		defer func() {
+			defer ginkgo.GinkgoRecover()
+			if errs := t.Cleanup(); len(errs) != 0 {
+				framework.Failf("errors in cleanup: %v", errs)
+			}
+		}()
+
+		ginkgo.By("creating service " + ns + "/" + serviceName + " with Service.Spec.IPFamily not set")
+		service := createService(t.ServiceName, t.Namespace, t.Labels, nil)
```

Comment:
why not use `jig.CreateTCPServiceWithPort()` ?  The Jig was intended to be the one way to do most service testing.

Reply:
@thockin thank you for the review! I can do this in a follow up PR.